ui-library-add-call-recording/api/onRecordingFileStatusUpdated.cs

```diff
@@ -0,0 +1,111 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using Microsoft.Azure.EventGrid;
+using System.Linq;
+using Newtonsoft.Json.Linq;
+using Microsoft.Azure.EventGrid.Models;
+using System.Collections.Generic;
+using Azure.Communication.CallingServer;
+using System.Text.RegularExpressions;
+using Contoso.Transfer;
+
+namespace Contoso
+{
+    public static class onRecordingFileStatusUpdated
+    {
+        [FunctionName("onRecordingFileStatusUpdated")]
+        public static async Task<IActionResult> Run(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", "post", Route = null)] HttpRequest req,

```

Comment:
can this be a "get" request if it needs the post body?

Reply:
Oh nvr mnd you've fixed this in your stacked PR 😊

Reply:
Yeah, slipped in this one.